README.md

```diff
@@ -313,6 +313,47 @@ async function run() {
 run();
 ```
 
+## Mint Compressed NFT
```

Comment:
Please explain what this is doing (e.g. generates a new collection if one doesn't exist) and briefly explain the different options.